### ISSUE TITLE
Help to implement the custom loss function!

### ISSUE THREAD

**Issue by Nishanthiruthayaraj** [sdv-dev/SDV#1165]
Hello, I saw your repository recently and found very interesting. I am also working on GAN using Pytorch for my thesis. I wanted to try your custom loss function in Pytorch. Could you please help me how can I implement this custom loss function ? Thank you very much. 

Custom Loss Function link:
https://github.com/sdv-dev/SDV/discussions/980

**Comment by npatki** [comment]
Hi @Nishanthiruthayaraj, the custom loss function you mention is a key innovation in the CTGAN model. Whenever you use CTGAN, you are already using the loss function.  

You can try CTGAN following this tutorial: [https://sdv.dev/SDV/user_guides/single_table/ctgan.html](https://sdv.dev/SDV/user_guides/single_table/ctgan.html)

Let me know if there are any other Qs!

**Comment by Nishanthiruthayaraj** [comment]
Hi **@npatki** thank you very much for the answer. I have one more question. I am training Vanilla GAN and my data is a continuous time series data. But I am not sure whether my model is performing well nor not. 

**Could you please also tell me how could I evaluate the generated data with real data?** Lets say, the array size of generated data is [200000, 19] and real data of size [300000, 19]. I couldn't find the apt solution for this type. Any help would move my thesis next. Thank you

**Comment by npatki** [comment]
Hi @Nishanthiruthayaraj, you can use the [SDMetrics library](https://docs.sdv.dev/sdmetrics/) to compare the real vs. synthetic data across a variety of dimensions. I would recommend applying the Quality Report, which is covered in the [Quickstart Guide](https://docs.sdv.dev/sdmetrics/getting-started/quickstart).

**Comment by Nishanthiruthayaraj** [comment]
Thank you very much @npatki

**Comment by npatki** [comment]
You're welcome -- and glad it helped! I'll close this issue off since we've answered the question. If you have any related topics, feel free to bring them up here. Otherwise, you are welcome to ask new questions by [filing another issue](https://github.com/sdv-dev/SDV/issues/new/choose) or [joining our Slack](https://bit.ly/sdv-slack-invite).
